src/test/java/org/opensearch/security/system_indices/SystemIndicesTests.java

```diff
@@ -198,14 +204,17 @@ public void testSearchWithSystemIndicesAsSuperAdmin() throws Exception {
 
 
     @Test
-    public void testSearchWithSystemIndicesAsAdmin() throws Exception {
```

Comment:
Did you mean to remove this test? I have not seen where it was moved to yet--if it is below just ignore this...

Reply:
I've renamed/refactored it to "testSearchWithSystemIndicesShouldFailAsAdmin" to be in line with the other tests / describe expected behaviour.